client/coral-framework/services/i18n.js

```diff
@@ -0,0 +1,93 @@
+const time = require('timeago.js');
```

Comment:
Our goal is to use es6 modules whenever we can. All nodejs `require` code should be changed to es6 modules.